salt/modules/mac_timezone.py

```diff
@@ -11,6 +11,8 @@
 
 # Import salt libs
 import salt.utils
+from salt.utils.mac_utils import execute_return_result, \
```

Comment:
We generally make these absolute. So import salt.utils.mac_utils and refer to each by their full-qualified name in the code.